svgelements/svgelements.py

```diff
@@ -5744,17 +5745,12 @@ def z_point(self):
         This behavior of Z is defined in svg spec:
         http://www.w3.org/TR/SVG/paths.html#PathDataClosePathCommand
         """
-        end_pos = None
         for segment in reversed(self._segments):
-            if isinstance(segment, Move):
-                end_pos = segment.end
-                break
-        if end_pos is None:
-            try:
-                end_pos = self._segments[0].end
-            except IndexError:
-                pass  # entire path is "z".
-        return end_pos
+            if isinstance(segment, (Move, Close)):
```

Comment:
Z is the destination of the last Move. Not the last move and close. This isn't a strong distinction but it's wrong to say the move or close, even though those *should* be the same point if this code has been validated.